library/Zend/ModuleManager/ModuleManager.php

```diff
@@ -52,6 +52,18 @@ class ModuleManager implements ModuleManagerInterface
     protected $modules = array();
 
     /**
+     * Childs dependency modules
+     *
+     * @var array
+     */
+    protected $childsModules = array();
+
+    /**
+     * @var ModuleEvent
+     */
+    protected $loadEvent;
```

Comment:
I don't quite understand why we need another copy of the ModuleEvent... Is it not already set in `$this->event` ?